file_handler.rb

```diff
@@ -0,0 +1,25 @@
+require 'json'
+class FileHandler
+  attr_accessor :path, :store
+
+  def initialize(path)
+    @path = path
+  end
+
+  def write_to_file(data)
+    file = File.open(@path, 'w')
+    # @store << data
+    file.write(data)
+    file.close
+  end
+
+  def read_from_file
+    return nil unless File.exist?(@path)
+
+    file = File.open(@path, 'r')
+    data = file.read
+    file.close
+
+    data
+  end
+end
```

Comment:
- [ ] I noticed that upon exit and restart of the application, all data previously inputted are lost. Therefore any new data I input will overwrite the previous one instead of being a continuation of the previous one. 

I also noticed that when I try to list the rentals and create a new rental upon restart of the application, the person `ID` can not be read or found. 

It would be nice to fix all these issues so that data is not lost and data is properly loaded and read upon restart of the application. 👍